kolibri/utils/conf.py

```diff
@@ -54,6 +54,9 @@
     "kolibri_exercise_perseus_plugin"
 ]
 
+#: Everything in this list is added to django.conf.settings.INSTALLED_APPS
+config['INSTALLED_APPS'] = DEFAULT_PLUGINS
```

Comment:
@rtibbles me too, if you open up an issue about explicitly disabling default plugins, that'd be great! :)